styles/main.css

```diff
@@ -59,37 +59,42 @@ main #useful-links a{
 footer {
     text-align: center;
     font-size: 12px;
-    margin: 0 0 0 3em;
+    margin: 0 0 3em 0;

```

Comment:
Wrong order: start at top and go clockwise from there (so bottom is the 3rd element)